CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## Unreleased
 
+## v3.0.0-RC1 (2025-04-14)
+
+### Breaking Changes
+- Remove deprecated triggers and functions support (#1524)
+- Changed default search `DIALECT` to `2` (#1516)
+- Implemented PSR-7 compatible stream abstraction (#1450)
+- Improved pipeline abstractions (#1438)
+- Improved connection handshake (#1431)
+
+### Added
+- Added Redis 7.4 `XREAD` and `HSCAN` interface changes (#1452)
+- Added support for transactions for clustered connections (#1497)
```

Comment:
This is a breaking change as well, right?

Reply:
Right, fixing

Reply:
1497 is mentioned in the v3.0.0-alpha1 changes already.

Reply:
Yeah, but it's a mistake, it's not included in alpha1 content. I moved it to correct place